test_runner/regress/test_storage_controller.py

```diff
@@ -3568,6 +3568,7 @@ def test_storage_controller_detached_stopped(
         tenant_id,
         shard_count=1,
     )
+    env.pageserver.quiesce_tenants()
```

Comment:
I had to add this since a detach could race with an attach and cancel the manifest upload, which would yield:

```
attach failed, setting tenant state to Broken: Cancelled
```

I'm assuming this is otherwise unproblematic and working as intended, but flagging it in case I'm missing something and we need to handle this case during attach.